core-services/prow/02_config/_plugins.yaml

```diff
@@ -1272,13 +1290,44 @@ lgtm:
 - repos:
   - openshift/extended-platform-tests
   review_acts_as_lgtm: true
+- repos:
+  - cloud-bulldozer/plow
+  review_acts_as_lgtm: true
 override:
   allow_top_level_owners: true
 owners:
   labels_blacklist:
   - approved
   - lgtm
 plugins:
+  cloud-bulldozer/plow:
+  - assign
+  - blunderbuss
+  - blockade
+  - bugzilla
+  - cat
+  - dog
+  - heart
+  - golint
+  - goose
+  - help
+  - hold
+  - label
+  - lgtm
+  - lifecycle
+  - override
+  - pony
+  - retitle
+  - shrug
+  - sigmention
+  - size
+  - skip
+  - trigger
+  - verify-owners
+  - owners-label
+  - wip
+  - yuks
+  - approve
```

Comment:
You are enabling plugins that handle approvals and reviews but you are not configuring Tide for automated merges (that act on approvals and reviews). Is that intentional?

Reply:
You do not need any of this if you only want periodics.